### ISSUE TITLE
No such property: versionInt when importing sdk in Android Studio 1.0.1 (Mac)

### ISSUE THREAD

**Issue by lgorse** [Kickflip/kickflip-android-sdk#18]
I'm getting an error when I import the Kickflip SDK into my project in Android Studio as a module.

The error occurs when I attempt to import the SDK as a module 
Here is the error:

    Error:(3, 0) No such property: versionInt for class: org.gradle.api.internal.project.DefaultProject_Decorated
Possible solutions: version

Information about my build.gradle:
```
apply plugin: 'com.android.application'
android {
    compileSdkVersion 19
    buildToolsVersion "19.0.0"
    defaultConfig {
        applicationId "com.example.lgorse.androidhive_test"
        minSdkVersion 18
        targetSdkVersion 19
        versionCode 1
        versionName "1.0"
    }
    buildTypes {
        release {
            minifyEnabled false
            proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
        }
        debug{
            debuggable true
        }
    }
}
dependencies {
    compile fileTree(dir: 'libs', include: ['*.jar'])
    compile 'com.android.support:appcompat-v7:21.0.3'
    compile project(':libstreaming')
    compile 'io.kickflip:sdk:1.1.1'
```

and here is the gradle build for the Kickflip SDK:

```
apply plugin: 'android-library'
versionInt=8
versionName="1.1.1"
android {
    compileSdkVersion 19
    buildToolsVersion "20.0"
    defaultConfig {
        minSdkVersion 18
        targetSdkVersion 19
        versionCode versionInt
        versionName versionName
    }
    buildTypes {
        release {
            runProguard true
            proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.txt'
        }
    }
    compileOptions {
        sourceCompatibility JavaVersion.VERSION_1_7
        targetCompatibility JavaVersion.VERSION_1_7
    }
}
configurations {
    jaxDoclet
    classpaths
}

dependencies {
    compile 'com.google.guava:guava:16.0'
    compile 'com.android.support:support-v4:19.+'
    // aws-sdk-android
    compile files('libs/aws-android-sdk-1.7.0-core.jar')
    compile files('libs/aws-android-sdk-1.7.0-s3.jar')
    // google-oauth-java-client
    compile files('libs/google-http-client-1.17.0-rc.jar')
    compile files('libs/google-http-client-android-1.17.0-rc.jar')
    compile files('libs/google-http-client-jackson2-1.17.0-rc.jar')
    compile files('libs/google-oauth-client-1.17.0-rc.jar')
    compile files('libs/gson-2.1.jar')
    compile files('libs/jsr305-1.3.9.jar')
    compile files('libs/protobuf-java-2.4.1.jar')
    // fulfilled by aws-android-sdk:
    //compile files('libs/jackson-core-2.1.3.jar')
    //compile files('libs/jackson-core-asl-1.9.11.jar')
    // m3u8 parser
    compile files('libs/m3u8parser-0.2.jar')
    // For Doclava JavaDoc
    jaxDoclet("com.google.doclava:doclava:1.0.5")
    //classpaths files('build/classes/release')
    classpaths fileTree(dir: 'build/classes/release').matching { include 'io/kickflip/sdk/Kickflip.java' }
    classpaths files(new File(System.getenv('ANDROID_HOME') + '/platforms/android-19/android.jar'))
}
// Maven publish task
// Uncomment this line to enable publishing to Maven Central. Requires maven
// credentials in gradle.properties
//apply from: 'https://raw.github.com/OnlyInAmerica/mvn-repo/master/maven.gradle'
// Generate JavaDoc
android.libraryVariants.all { variant ->
    task("generate${variant.name}Javadoc", type: Javadoc) {
        source = android.sourceSets.main.java
        title = null
        options.docletpath = configurations.jaxDoclet.files.asType(List)
        options.classpath = configurations.classpaths.files.asType(List)
        options {
            doclet "com.google.doclava.Doclava"
            bootClasspath new File(System.getenv('JAVA_HOME') + "/jre/lib/rt.jar")
            addStringOption "warning", '110'  // Allow undocumented types
            addStringOption "overview", "/Users/davidbrodsky/Code/android/kickflip-android-example/submodules/kickflip-android-sdk/sdk/src/javadoc-overview.html"
            addStringOption "public"
            addStringOption "hdf project.name", "Kickflip Android SDK"
            addStringOption "federate android", "http://d.android.com/reference"
            addStringOption "federationxml android", "http://doclava.googlecode.com/svn/static/api/android-10.xml"
        }
    }
}
```

**Comment by lgorse** [comment]
OK the reason is that your opening variable definition should be prefixed with "def"

**Comment by OnlyInAmerica** [comment]
Thanks for the report! This is addressed in [8600e94](https://github.com/Kickflip/kickflip-android-sdk/commit/8600e9494ad8352472100d165f767f797a96aadc)
